src/datasets/builder.py

```diff
@@ -790,8 +806,9 @@ def _download_and_prepare(self, dl_manager, verify_infos, **prepare_split_kwargs
 
             try:
                 # Prepare split will record examples associated to the split
-                self._prepare_split(split_generator, **prepare_split_kwargs)
+                self._prepare_split(split_generator, file_format=file_format, **prepare_split_kwargs)
             except OSError as e:
+                raise
```

Comment:
This was added for debugging? It makes the raise below unreachable

Reply:
yea sorry, forgot to remove it